include/g_compensation/g_compensator_class.h

```diff
@@ -60,6 +60,14 @@ class GCompensator
             buffer_size_ = 50; //default value
             ROS_WARN("Continuing with default buffer_size '50'.");
         };
+
+        bool negate_wrench_flag_;
+        if (!rosparam_shortcuts::get("g_compensator", n_static_params, "negate_wrench", negate_wrench_flag_))
+        {
+            negate_wrench_flag_ = false; //default value
+            ROS_WARN("Continuing with not negating the wrench received per default.");
+        };
+        negate_wrench_ = int(negate_wrench_flag_) * -2 + 1;
```

Comment:
Nice computation, but personally, I'd prefer more obvious semantics like
`negate_wrench_ = negate_wrench_flag ? -1 : 1;`

Reply:
Yes, I am not a C++ developer. I changed it in [cceae46](https://github.com/ipa320/g_compensator/pull/4/commits/cceae46ea82d89a25c0439824de75746f6344877).